jobbergate-api/tests/apps/test_services.py

```diff
@@ -74,16 +74,14 @@ async def test_create__success(
         """
         Test that the ``create()`` method successfully creates an instance of the served model.
         """
-        with time_frame() as window:
-            instance = await dummy_crud_service.create(
-                name="test-name",
-                description="test-description",
-                owner_email=tester_email,
-            )
+
+        instance = await dummy_crud_service.create(
+            name="test-name",
+            description="test-description",
+            owner_email=tester_email,
+        )
 
         assert instance.owner_email == tester_email
-        assert instance.created_at in window
-        assert instance.updated_at in window
```

Comment:
This is best for most test cases. For some reason, I get a very occasional off-by-one error for the window test. I have tried to isolate it, but failed. I try to limit the window tests to only methods where it's important to verify that the timestamps are being used correctly.

Reply:
Yeah, it used to fail randomly, and it broke when I changed the time control to `pendulum` to facilitate the tests. Since it was just used in this test, I removed it. We can always go back and refactor it to work with pendulum, at least it would be more precise.